easybuild/easyconfigs/g/GUDHI/GUDHI-3.4.1-cmake_test.patch

```diff
@@ -0,0 +1,163 @@
+diff -Nru gudhi.3.4.1.orig/python/CMakeLists.txt gudhi.3.4.1/python/CMakeLists.txt
```

Comment:
Please add a comment at the top of the patch including your name and the reason for the patch i.e. which problem it is solving.

Reply:
Sure!